pkg/kritis/metadata/containeranalysis/containeranalysis.go

```diff
@@ -72,11 +72,19 @@ func (c ContainerAnalysis) GetVulnerabilities(containerImage string) ([]metadata
 }
 
 // GetAttestation gets AttesationAuthority Occurrences for a specified image.
-func (c ContainerAnalysis) GetAttestations(containerImage string) ([]*containeranalysispb.Occurrence, error) {
-	return c.fethcOccurrence(containerImage, AttestationAuthority)
+func (c ContainerAnalysis) GetAttestations(containerImage string) ([]metadata.PgpAttestation, error) {
+	occs, err := c.fetchOccurrence(containerImage, AttestationAuthority)
+	if err != nil {
+		return nil, err
+	}
+	pgpAttestations := make([]metadata.PgpAttestation, len(occs))
```

Comment:
Variable names in Go should be short rather than long. This is especially true for local variables with limited scope. Prefer c to lineCount. Prefer i to sliceIndex.

Maybe just "p" here? The variable lifetime is very short, and it's already clear from the context that this is a slice of PgpAttestations.

Reply:
thanks. The reason why i had "Pgp" Vs "PGP" was because the containeranalysis proto has it as PgpSignedAttestation.
I can change it to fix it.

Reply:
Ahh, it's a proto. Proto's always have braindead capitalization :(